pinot-distribution/pom.xml

```diff
@@ -199,6 +199,10 @@
                   <pattern>org.apache.parquet</pattern>
                   <shadedPattern>shaded.org.apache.parquet</shadedPattern>
                 </relocation>
+                <relocation>
```

Comment:
not relevant to this pr, shall we change the shaded pattern to something like shaded.pinot.xxx ?
We can have another PR for this

Reply:
Already in local, I was about to push that later in a different PR.